test/correctness/gpu_reuse_shared_memory.cpp

```diff
@@ -85,32 +82,26 @@ int pyramid_test() {
 
 int inverted_pyramid_test() {
     const int levels = 6;
-    const int size_x = 8*16*4;
-    const int size_y = 8*16*4;
+    const int size_x = 8 * 16 * 4;
+    const int size_y = 8 * 16 * 4;
 
     Var x, y, z, yo, yi, yii, xo, xi, xii, thread_x, thread_y;
 
     std::vector<Func> funcs(levels);
 
     funcs[0](x, y) = 1;
     for (int i = 1; i < levels; ++i) {
-        funcs[i](x, y) = funcs[i-1](x/2, y);
+        funcs[i](x, y) = funcs[i - 1](x / 2, y);
     }
 
-    funcs[levels-1].compute_root()
-        .tile(x, y, xi, yi, 64, 64)
-        .gpu_blocks(x, y)
-        .tile(xi, yi, xii, yii, 16, 16)
-        .gpu_threads(xi, yi);
-    for (int i = levels-2; i >= 0; --i) {
-        funcs[i].compute_at(funcs[levels-1], x)
-            .tile(x, y, xi, yi, 4, 4)
-            .gpu_threads(xi, yi);
+    funcs[levels - 1].compute_root().tile(x, y, xi, yi, 64, 64).gpu_blocks(x, y).tile(xi, yi, xii, yii, 16, 16).gpu_threads(xi, yi);
```

Comment:
here too, though the original looks correct, so I'm not sure what clang-format did.